hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -54,13 +54,23 @@
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientMapBasicTest {
+    private static final String KEY = "key";
+    private static final String OLD_VALUE = "oldValue";
+    private static final String NEW_VALUE = "newValue";
 
-    static HazelcastInstance client;
-    static HazelcastInstance server;
+    private static HazelcastInstance client;
+    private static HazelcastInstance server1;
+    private static HazelcastInstance server2;
+    private static HazelcastInstance server3;
+
+    private String randomMapName = randomString();
```

Comment:
It's not because the tests are all designed to use a unique map and don't change anything in cluster topology. They just change and test data on a map level. This worked before and was not changed by me.